### PR TITLE
Support for WP8 Ion

### PR DESCRIPTION
**Pull request by teobugslayer**
See http://teampulse.telerik.com/view#item/262237

### CODE REVIEW

File: mobile/app-identifier.ts [hunk 1]
```diff
@@ -193,6 +193,32 @@ export class IOSNativeScriptCompanionAppIdentifier implements Mobile.IAppIdentif
 
 }
 
+export class WP8CompanionAppIdentifier implements Mobile.IAppIdentifier {
+	get appIdentifier(): string {
+		return "";
+	}
+
+	get deviceProjectPath(): string {
+		return "";
```
Comment (rosen-vladimirov) [inline mobile/app-identifier.ts]: Is this really empty string? Or maybe it is not used at all when it's inside the companion app?

**Comment by rosen-vladimirov** [comment]
:+1:  after checking my comment (not sure if it's relevant at all)

File: mobile/app-identifier.ts [hunk 2]
```diff
@@ -193,6 +193,32 @@ export class IOSNativeScriptCompanionAppIdentifier implements Mobile.IAppIdentif
 
 }
 
+export class WP8CompanionAppIdentifier implements Mobile.IAppIdentifier {
+	get appIdentifier(): string {
+		return "";
```
Comment (Fatme) [inline mobile/app-identifier.ts]: Why this is empty string? It is strange to me wp8 companion app identifier to be empty string...

**Comment by rosen-vladimirov** [comment]
:+1:

**Pull request merged**